src/ucp/core/ucp_context.c

```diff
@@ -157,6 +157,12 @@ static ucs_config_field_t ucp_context_config_table[] = {
    "protocol. Lanes slower than the specified ratio will not be used.",
    ucs_offsetof(ucp_context_config_t, multi_lane_max_ratio), UCS_CONFIG_TYPE_DOUBLE},
 
+  {"MULTI_PATH_RATIO", "1",
```

Comment:
maybe make it simpler, just define "by-device" or "by-path"?
it would allow simplifying transport select logic

Reply:
Imo, this config is more flexible.  What if we have CX5 + CX7 devices on the system? Or IB + low-speed Eth?

Reply:
eventually we want this to be exposed from UCT layer, so not sure having this in UCP is so useful
IMO UCP better have a more "strict"/"forceful" config that easy to understand how they affect the end result, and UCT could expose fine tunings of its performance estimations (similar to shared memory BW, PCIe BW , etc)